src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/SaySomethingForm/TextForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button } from '@blueprintjs/core'
+import { Intent } from '@blueprintjs/core'
+import classnames from 'classnames'
+import React, { FC, Fragment } from 'react'
+import TextareaAutosize from 'react-autosize-textarea'
+import SmartInput from '~/components/SmartInput'
+
+import style from '../style.scss'
+
+import { FormState } from './index'
+
+interface Props {
+  formState: FormState
+  dispatchForm: any
+}
+
+const SaySomethingFormText: FC<Props> = props => {
+  const { formState, dispatchForm } = props
+  const { text, variations } = formState
+
+  const updateVariations = (value, index) => {
+    const newVariations = formState.variations
+
+    newVariations[index] = value
+
+    dispatchForm({ type: 'updateData', data: { value: newVariations, field: 'variations' } })
+  }
+
+  const deleteVariation = index => {
+    variations.splice(index, 1)
+    dispatchForm({ type: 'updateData', data: { value: variations, field: 'variations' } })
+  }
+
+  const handleKeyDown = e => {
+    if ((e.ctrlKey || e.metaKey) && e.keyCode === 65) {
+      e.target.select()
+    }
+  }
+
+  return (
+    <Fragment>
+      <label className={style.fieldWrapper}>
+        <span className={style.formLabel}>Message*</span>
+        <TextareaAutosize
+          className={style.textarea}
+          onKeyDown={handleKeyDown}
+          value={text}
+          rows={1}
+          maxRows={4}
+          onChange={e => dispatchForm({ type: 'updateData', data: { field: 'text', value: e.currentTarget.value } })}
+        ></TextareaAutosize>
+      </label>
+      <div className={style.fieldWrapper}>
+        <span className={style.formLabel}>Alternates</span>
+        {variations &&
+          variations.map((variantion, index) => (
+            <div key={index} className={style.innerWrapper}>
+              <SmartInput
+                value={variantion}
+                onChange={value => updateVariations(value, index)}
+                className={classnames(style.textarea, style.multipleInputs)}
+                isSideForm
+                singleLine={false}
+              >
+                <Button
+                  icon="trash"
+                  minimal
+                  small
+                  intent={Intent.DANGER}
+                  onClick={() => deleteVariation(index)}
+                ></Button>
+              </SmartInput>
+            </div>
+          ))}
+        <Button onClick={() => dispatchForm({ type: 'addVariation' })} className={style.addContentBtn} large={true}>
+          Add Alternates
+        </Button>
+      </div>
+    </Fragment>
+  )
+}
```

Comment:
I am not sure about the way it has been implemented. We use `react-jsonschema-form` to handle content types, which allows us to define the complete schema of the element in the `text.js` content type file. 

If we go this way, then content type files will be mostly useless, and we'd need to remove the jsonSchema and uiSchema part.